samples/subsys/ipc/ipc_service/remote/src/main.c

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr.h>
+#include <drivers/ipm.h>
+#include <sys/printk.h>
+#include <device.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+
+#include <ipc/ipc_service.h>
+
+
+#define APP_TASK_STACK_SIZE (1024)
+K_THREAD_STACK_DEFINE(thread_stack_1, APP_TASK_STACK_SIZE);
+K_THREAD_STACK_DEFINE(thread_stack_2, APP_TASK_STACK_SIZE);
+
+static struct k_thread thread_data_1, thread_data_2;
+static volatile unsigned int received_data_1, received_data_2;
+
+static K_SEM_DEFINE(bound_ept1_sem, 0, 1);
+static K_SEM_DEFINE(bound_ept2_sem, 0, 1);
+
+static K_SEM_DEFINE(data_rx1_sem, 0, 1);
+static K_SEM_DEFINE(data_rx2_sem, 0, 1);
+
+static struct ipc_ept *ept_1, *ept_2;
```

Comment:
I am not sure if our coding standard allows that.
I would declare those variables in separated lines also in other files